src/useQuery.ts

```diff
@@ -63,12 +66,22 @@ export function useQuery<TData = any, TVariables = OperationVariables>(
     context,
     metadata,
     variables,
-    fetchPolicy,
+    fetchPolicy: actualCachePolicy,
     errorPolicy,
     fetchResults,
   }: QueryHookOptions<TVariables> = {}
 ): QueryHookResult<TData, TVariables> {
   const client = useApolloClient();
+  const ssrManager = useContext(SSRContext);
+
+  // Modify fetch policy for SSR mode.
+  const fetchPolicy =
+    ssrManager &&
```

Comment:
I did it that way first, but then i changes it because of skip part (can’t add link from mobile 😕) where this cariable will be useless and these two variables will be used again. Also ssrManager.register has to be used with ! because it’s nullable and typescript isn’t smart enough to assert type.